issue_01/script.js

```diff
@@ -0,0 +1,4 @@
+const ul = document.getElementsByTagName('ul');
```

Comment:
IMO
If the element to be retrieved is predetermined, it is ideal to give it an id and retrieve it.

Reply:
Thank you.
I have given the ul element an id and changed the method.
[5e283ad](https://github.com/yoshikawa-akr/js-lessons/pull/1/commits/5e283ad7096e5d6acd94d6e58a649be06b61d0e0)